ptu/global.ini

```diff
@@ -19087,10 +19087,10 @@ Journal_System_Stanton_Stanton2_Covalex_InvestigatorPreliminaryReport_Author=Cov
 Journal_System_Stanton_Stanton2_Covalex_InvestigatorPreliminaryReport_Title=A.Skenning - Wiederhergestellte Daten
 Journal_System_Stanton_Stanton2_Covalex_TechnicianDoorCodes_Author=Covalex Investigation - Unterkunft Code-Liste
-Journal_System_Stanton_Stanton2_Covalex_TechnicianDoorCodes_Body=An: Ava Skenning\nVon: Dennis Gethart\nBetreff: Unterkunft-Codes\n21.09.2945 16:53SET\n----------------------------\n\nAva,\n\nich habe die Türcodes vom SysAdmin-Terminal abgerufen und getestet. Hier ist die Liste:\n\nUnterkunft 1: Hammell, Scott = 1170\nUnterkunft 2: Biollo, Ross = 3633\nUnterkunft 3: Ososky, Mel = 1352 (*** Hinweis: Dieser Code scheint veraltet zu sein. Hat nicht funktioniert an der Tür)\nUnterkunft 4: Santos, Kyomi = 9898\nUnterkunft 5: Theolone, Nico = 1318\nUnterkunft 7: Besser, Claudia = 7871\nUnterkunft 8: Temp Housing = 2231\n\nUnd hier ist genau das, was Sie wollten:\n\nUnterkunft 6: Ward, Darnell = 6682\n\n\nLassen Sie mich wissen, wenn Sie noch etwas brauchen,\n\nDennis
+Journal_System_Stanton_Stanton2_Covalex_TechnicianDoorCodes_Body=An: Ava Skenning\nVon: Dennis Gethart\nBetreff: Unterkunft-Codes\n21.09.2945 16:53SET\n----------------------------\n\nAva, ich habe die Türcodes vom SysAdmin-Terminal abgerufen und getestet. Hier ist die Liste:\n\n• Unterkunft 1: Hammell, Scott = 1170\n• Unterkunft 2: Biollo, Ross = 3633\n• Unterkunft 3: Ososky, Mel = 1352 (*** Hinweis: Dieser Code scheint veraltet zu sein. Hat nicht funktioniert an der Tür)\n• Unterkunft 4: Santos, Kyomi = 9898\n• Unterkunft 5: Theolone, Nico = 1318\n• Unterkunft 7: Besser, Claudia = 7871\n• Unterkunft 8: Temp Housing = 2231\n\nUnd hier ist genau das, was Sie wollten:\n\n• Unterkunft 6: Ward, Darnell = 6682\n\n\nLassen Sie mich wissen, wenn Sie noch etwas brauchen,\n\nDennis
```

Comment:
Ist die Reihenfolge der Unterkünfte gewollt so komisch?

Reply:
Ist tatsächlich so auch in der original datei

Reply:
Dann lassen wirs^^